common-money.lic

```diff
@@ -160,4 +160,16 @@ module DRCM
       end
     end
   end
+
+  def deposit_coins_from_specific_bank?(keep_copper, settings, town)
```

Comment:
How does this function differ from `deposit_coins`?

Reply:
The only difference is no @hometown function is in it, just like get_money_from_specific_bank? 

I put in the minimize_coins line because I found it deposited all my money in my hometown. Or it did on early runs. I need to test again if I need it or not. Def `deposit_coins` assumes hometown and even sets it explicitly. I could see about combining the two different defs, `get_money_from_bank` with `get_money_from_specific_bank?` and `deposit_coins` with `get_money_from_specific_bank?`, this week or so. 

Now that I am really looking at it, let me try both in testing and see what happens. :) Going to change this back to draft.